### PR TITLE
Solution: Fix Job Broadcaster :delete

### PR DESCRIPTION
**Pull request by maennchen**

### CODE REVIEW

**Comment by maennchen** [comment]
@barthez I added a test for it. (A test if it actually removed the job should've been in here from the start anyways.)

File: test/quantum/job_broadcaster_test.exs [hunk 1]
```diff
@@ -84,6 +84,10 @@ defmodule Quantum.JobBroadcasterTest do
         TestScheduler.delete_job(broadcaster, active_job.name)
 
         assert_receive {:received, {:remove, ^active_job_name}}
+
+        refute Enum.any?(TestScheduler.jobs(broadcaster), fn {key, _} ->
```
Comment (barthez) [inline test/quantum/job_broadcaster_test.exs]: what about:
```elixir
refute %{^active_job_name => _} = TestScheduler.jobs(broadcaster)
```
Same below.

Comment (maennchen) [reply test/quantum/job_broadcaster_test.exs]: Because of BC reasons, this is sadly not a map, but a list containing a tuple of name & job.

**Comment by barthez** [comment]
Ah. Right. Got it now.

**Pull request merged**